### PR TITLE
Features/add the full project

### PR DESCRIPTION
**Pull request by Alzubair98**
the full project is ready to deploy

### CODE REVIEW

**Comment by Alzubair98** [comment]
i have some linter error in webhint 
and i faced that in the fourth week its because of the node.js version
i hope you consider this

**Comment by ahmetbozaci** [comment]
# STATUS: INVALID  ❌ 


### Hello @Alzubair98  ✋🏼 
#### You have submitted a project, that has linter errors. Kindly fix the linter errors then send it for review again.
#### Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏


### Cheers and Happy coding! 💻 👏
coding


___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._

_Invalid Code Review Request **does not count** into the code reviews limit._

___

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_

[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

**Comment by ahmetbozaci** [comment]
> i have some linter error in webhint and i faced that in the fourth week its because of the node.js version i hope you consider this

Kindly fix the error. Code reviewers won't review if you won't fix it.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,49 @@
...
+
+- Major languages
+- Frameworks
+- Technologies used
+
+## Live Demo (if available)
+
+[Live Demo Link](https://alzubair98.github.io/first-capstone-project/)
+
+To get a local copy up and running follow these simple example steps.
```
Comment (gemmen29) [inline README.md]: - [x] Kindly edit the info of the README file `Project Name`, `Description`, `Build with` and `steps to get a local copy` if exist.
- [x] [OPTIONAL] Please add an image for the website in your README in order to make your README more professional.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,49 @@
...
+
+## Authors
+
+👤 **Author1**
```
Comment (gemmen29) [inline README.md]: - [x] Kindly update `Author1` with your name instead.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,49 @@
...
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Alzubair98/Hello---micro/issues).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
```
Comment (gemmen29) [inline README.md:44]: - [x] Kindly add an acknowledgment to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

File: README.md [hunk 4]
```diff
@@ -1,2 +1,49 @@
-# first-capstone-project
-this is the first capstone project for the first module in microverse
+
```
Comment (gemmen29) [inline README.md:1]: - [x] Kindly add a presentation video as per this [requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/capstone/html_capstone.md#:~:text=Once%20you%20have,very%20proud%20of).

File: about.html [hunk 5]
```diff
@@ -0,0 +1,148 @@
...
+                <img class="microsoft-img" src="images/microsoft.png" alt="microsoft-img">
+                <img class="izmer-img" src="images/izmer.png" alt="izmer-img">
+                <img class="aydin-img" src="images/aydin.png" alt="aydin-img">
+            </div>
+        </div>
+
+        <div class="mobile-fifth-section">
+            <img class="desktop-nav-logo altin-bas-logo" src="images/altinlogo.jpg" alt="altinbas-logo">
+            <p class="paragraph-fourth-section">2020 Altinbas University Confrenece Some Rights Reserved</p>
+        </div>
```
Comment (gemmen29) [inline about.html]: - [x] Please make the partner section disappear and change the background to dark on the `about` page to match the Behance design.

| Your design | Behance design |
| --- | --- |
| image | image |

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @Alzubair98, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: about.html (hunk 5 above)
Comment (Alzubair98) [reply about.html]: done

File: README.md (hunk 3 above)
Comment (Alzubair98) [reply README.md:44]: done

File: README.md (hunk 1 above)
Comment (Alzubair98) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (Alzubair98) [reply README.md]: done

File: README.md (hunk 4 above)
Comment (Alzubair98) [reply README.md:1]: done

**Review by OmarRamoun — APPROVED** [review]
# 🟢 Project Approved! 🏆

## Hi @Alzubair98 👋

Your project is complete! ✔️ 🏁

Finally, **you did it**! 🎉 👏. Your code was very professional 💯 and I can see you have a great chance to be a Great Software Engineer ⛑️.

There is nothing else to say other than... _it's time to merge it_ :shipit:
Congratulations! 🎉. And Welcome to Microverse 👋.

> **Note:** You can now take a break even if short before you move on to the next project. Happy 😄 Coding 🖥️! and Have a nice day 🌞.

## Highlights 🌟

- ✔️ Descriptive commit messages
- ✔️ Descriptive PR description
- ✔️ No linter Errors
- The code is top-notch! 💯 and have a huge potential.

Cheers and Happy coding!👏👏👏

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@omarramoun](https://github.com/omarramoun)
- This is My **Twitter**: [@omarramoun](https://twitter.com/omarramoun)
- This is My **Slack**: [Ramoun](https://slack.com)
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**